### PR TITLE
BLS Region SELECT added

### PR DESCRIPTION
**Pull request by mistergone**
This PR adds a `SELECT` to the expenses section which allows the user to choose a BLS region. Changing this `SELECT` will reset the `INPUT`s in the expenses section to the values returned by our API. Initially, when a program loads, this `SELECT` will be set to the region associated with the program

**Note!** This PR does not include proper styling for the `SELECT` (it's too wide), and the expenses section has yet to be re-ordered. Those change are in a separate task and will be completed later.
## Additions
- Added the `#bls-region-select` element
- Added a handler to `expenses-view` for the `SELECT` element
- Functional test added for BLS region changing
## Testing
- Functional and unit tests are all passing!
## Review
- @marteki and @amymok  and @higs4281 and anyone who likes BLS data!
- Pull it down, run `gulp` (`npm install` shouldn't be necessary), and check it out!
## Screenshots

screen shot 2016-06-06 at 3 11 34 pm
## Checklist
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code can be automatically merged (no conflicts)
- [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
- [x] Passes all existing automated tests
- [x] New functions include new tests
- [x] New functions are documented (with a description, list of inputs, and expected output)
- [x] Placeholder code is flagged
- [x] Visually tested in supported browsers and devices 
- [x] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)
## Animated GIF

(for @ascott1 ) Ladies and gentlemen, **The Selecter!!!**

### CODE REVIEW

File: src/disclosures/js/index.js [hunk 1]
```diff
@@ -49,9 +49,7 @@ var app = {
 
               // Update expenses model bases on region and salary
               region = schoolValues.BLSAverage.substr( 0, 2 );
-              salary = schoolValues.medianSalary;
-              expensesModel.resetCurrentValues( region, salary );
-              expensesView.updateView( getExpenses.values() );
+              $( '#bls-region-select' ).val( region ).change();
```
Comment (marteki) [inline src/disclosures/js/index.js:50]: Please fix/remove the following variables that are throwing new `lint` warnings for this file:

> 9:5   warning  'getExpenses' is defined but never used
>   38:19  warning  'salary' is defined but never used

File: src/disclosures/js/views/expenses-view.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var getExpenses = require( '../dispatchers/get-expenses-values' );
+var getFinancials = require( '../dispatchers/get-financial-values' );
```
Comment (marteki) [inline src/disclosures/js/views/expenses-view.js]: Please remove this line, as it's throwing the following new `lint` warning:

>   4:5  warning  'getFinancials' is defined but never used

**Comment by marteki** [comment]
The selector is working well! I pointed out some new `lint` warnings that could easily be addressed to keep our code nice and tidy. Fix those, and this gets a 👍  from me for merge.

NOTE: I'm also seeing some weird behavior again around manually changing the numbers in the expenses form fields, where dollar signs aren't re-appearing on values. 😫

**Comment by mistergone** [comment]
The latest commits should fix the linting issues (sorry!) and I implemented a fix that should address the issues you're seeing with the expenses input fields.

**Pull request merged**